core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -6325,11 +6328,15 @@ components:
     WorkspaceBucketOptions:
       required:
         - requesterPays
+        - location
```

Comment:
Now that location is a required response in BucketOptions, we need to check/update rawls producer contract tests so as not to break them.

Reply:
Do you mean that for other services where Rawls is the provider, their contracts will have to be updated once the new Rawls version is incorporated? Or do I have to make a manual change in Rawls somewhere?

Reply:
I meant the provider state should be updated if the responses are changing. 
On quick review of [Rawls provider tests](https://github.com/broadinstitute/rawls/blob/develop/pact4s/src/test/scala/org/broadinstitute/dsde/rawls/provider/RawlsProviderSpec.scala), there doesn't seem to be a provider test for Bucket Options. Therefore there may be no need for you to do anything!